src/links/links.controller.ts

```diff
@@ -8,12 +8,14 @@ import {
   Param,
   Patch,
   Post,
+  Req,
   ValidationPipe,
 } from '@nestjs/common';
 import { LinksService } from './links.service';
-import { Link, Prisma } from '@prisma/client';
 import { CreateLinkDto } from './dto/create-link.dto';
 import { UpdateLinkDto } from './dto/update-link.dto';
+import { Request } from 'express';
+import { JwtUserPayload } from 'src/types/JwtUserPayload';
```

Comment:
**suggestion**: The endpoint path `/generate` is clear, but consider documenting it for client developers.